frontend/helpers/next-helpers.ts

```diff
@@ -7,6 +7,7 @@ import { clearCache } from '@lib/cache';
 import { CACHE_DISABLED } from '@config/env';
 
 export function serverSidePropsWrapper<T extends { [key: string]: any }>(
+   pageName: string,
    getServerSidePropsInternal: GetServerSideProps<T>
 ): GetServerSideProps<T> {
    return async (context) => {
```

Comment:
> next.js doesn't have any way that I've seen to get the pathname like
"product/[handle]" Though there's a GH issue to add this feature.

Is [`context.resolvedUrl`](https://nextjs.org/docs/api-reference/data-fetching/get-server-side-props#context-parameter) not what you're looking for here? Like this:

https://github.com/iFixit/react-commerce/blob/66ae2a62c117aa219321e69a981a8dbada230958/packages/helpers/nextjs.tsx#L3-L6

Reply:
Sorry, no, I want a canonicalized route name. Effectively the name of the file. Turns out a human generated name is probably better anyway.